result/bmpprefix.yaml

```diff
@@ -0,0 +1,652 @@
+components:
+  schemas:
+    BmpPrefix.State.Request:
+      description: >-
+        The request to retrieve BMP server learned BGP peer prefix information.
+      type: object
+      properties:
```

Comment:
I think array of BMP server names is a must , anything else would be maybe per BMP server name if at all.. not sure we need more depth of control